net/Socket.cpp

```diff
@@ -550,6 +619,25 @@ int SocketPoll::poll(int64_t timeoutMaxMicroS)
                 // removed in a callback
                 ++itemsErased;
             }
+            else if (!_pollSockets[i]->isOpen())
```

Comment:
Ok - so; this chunk alone seems plausible, but potentially controversial / regression causing. Can it be a separate commit - so it is bisectable; ideally along with a clear rationale for that ?

Reply:
this is in its single commit, compile clean on its own in this patch series, i.e. https://github.com/CollaboraOnline/online/pull/9916/commits/7db79302b7a4ea310c4defe3009e6818d43c3dc5
hence it is bisectable.